devlab/tests/filtering_utils.py

```diff
@@ -77,3 +112,25 @@ def get_vm_fip(vm):
                     return addr['addr']
         raise RuntimeError('VM with name {} and id {} doesnt have fip'.format(
             vm.name, vm.id))
+
+    def wait_until_vm_accessible_via_ssh(self, ip_addr):
+        for i in range(VM_ACCESSIBILITY_ATTEMPTS):
+            try:
+                self.execute_command_on_vm(ip_addr, 'pwd')
+                break
+            except RuntimeError:
+                time.sleep(1)
+        else:
+            msg = 'VM with ip "{}" is not accessible via ssh after {} attempts'
+            raise RuntimeError(msg.format(ip_addr, VM_ACCESSIBILITY_ATTEMPTS))
+
+    @staticmethod
+    def open_ssh_port_secgroup(client, tenant_id):
+        sec_grps = client.get_sec_group_id_by_tenant_id(tenant_id)
+        for sec_gr in sec_grps:
+            try:
+                client.create_security_group_rule(
+                    sec_gr, tenant_id, protocol='tcp', port_range_max=22,
+                    port_range_min=22, direction='ingress')
+            except NeutronClientException:
+                pass
```

Comment:
I don't think you should pass this exceptions, it's pretty critical. If you want to handle only duplicate entry - please choose another/specific Neutron Exceprtion. 

Thank you.